third_party/3/pandas/core/series.pyi

```diff
@@ -146,4 +152,5 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
     def notnull(self) -> Series: ...
     def dropna(self, axis: OneDimensionalAxisOption = ..., inplace: bool = ..., how: Optional[Any] = ...) -> Optional[Series]: ...
     def to_timestamp(self, freq: Optional[str] = ..., how: TimestampMethod = ..., copy: bool = ...) -> Series: ...
-    def to_period(self, freq: Optional[str] = ..., copy: bool = ...) -> Series: ...
\ No newline at end of file
+    def to_period(self, freq: Optional[str] = ..., copy: bool = ...) -> Series: ...
+    def compare(self, other: Series, align_axis: Union[str, int] = ..., keep_shape: bool = ..., keep_equal: bool = ...) -> FrameOrSeries: ...
```

Comment:
There's the `AxisOption` in _typing which covers the type of `align_axis`

Reply:
Right, how could I not notice them, they are used just few lines above. Thanks!